### PR TITLE
Train Issue Fix

### PR DESCRIPTION
**Pull request by AshishMahendra**

### CODE REVIEW

File: jaseci_kit/jaseci_kit/encoders/utils/models.py [hunk 1]
```diff
@@ -176,16 +176,53 @@ def __init__(self, config, *inputs, **kwargs):
...
+        return loss
 
     def forward(self, context_input_ids=None, context_input_masks=None,
                 candidate_input_ids=None, candidate_input_masks=None,
                 labels=None, get_embedding=None, mode="train", pooling="mean"):
         # only select the first candidate (whose lbl==1)
-        if labels is not None:
-            candidate_input_ids = candidate_input_ids[:, 0, :].unsqueeze(1)
-            candidate_input_masks = candidate_input_masks[:, 0, :].unsqueeze(1)
+        # if labels is not None:
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/encoders/utils/models.py:220]: remove unncessary code please.

File: jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py [hunk 2]
```diff
@@ -31,17 +31,23 @@ def __init__(self, contexts, candidates, context_transform,
...
-                        'context': None,
-                        'candidates': [],
-                        'labels': []
-                    }
+                group = {
+                    'context': None,
+                    'candidates': [],
+                    'labels': []
+                }
+                # below code is to combine positive and negetive sample
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py:39]: Same here.

File: jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py [hunk 3]
```diff
@@ -31,17 +31,23 @@ def __init__(self, contexts, candidates, context_transform,
...
+                #     group = {
+                #         'context': None,
+                #         'candidates': [],
+                #         'labels': []
+                #     }
                 group['candidates'].append(cand)
                 group['labels'].append(lbl)
                 group['context'] = [text]
-            if len(group['candidates']) > 0:
+            # if len(group['candidates']) > 0:
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py:50]: is this right?

File: jaseci_kit/jaseci_kit/encoders/utils/evaluate.py [hunk 4]
```diff
@@ -31,7 +31,7 @@ def get_inference(model, tokenizer, contexts, candidates, train_config):
         batch = tuple(t.to(train_config['device']) for t in batch)
         context_token_ids_list_batch, context_input_masks_list_batch,\
             candidate_token_ids_list_batch, candidate_input_masks_list_batch,\
-            labels_batch = batch
+            _ = batch
```
Comment (ypkang) [inline jaseci_kit/jaseci_kit/encoders/utils/evaluate.py:34]: what's going on here?

**Review by ypkang — CHANGES_REQUESTED** [review]
just have some minor comments.

File: jaseci_kit/jaseci_kit/encoders/utils/models.py (hunk 1 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/encoders/utils/models.py:220]: this code was used when, the positive and negative sample for 1 datapoint is combined

File: jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py (hunk 2 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py:39]: this code was used when, the positive and negative sample for 1 datapoint is combined

File: jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py (hunk 3 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/encoders/utils/tokenizer.py:50]: yes

File: jaseci_kit/jaseci_kit/encoders/utils/evaluate.py (hunk 4 above)
Comment (AshishMahendra) [reply jaseci_kit/jaseci_kit/encoders/utils/evaluate.py:34]: This was when i merged the branch and was trying to push with a change, the labels are not used for evaluations

**Review by ypkang — APPROVED** [review]

**Pull request merged**